Create Transformer_current_SMD_8.4x7.2mm.kicad_mod

evanshultz opened the pull request:
This is an SMT current sense transformer footprint that is rather common for currents at 10-20A. See the following datasheets:
1. http://products.pulseelex.com/files/product_files/PC2005_47.pdf
2. http://products.pulseelex.com/files/product_files/SPM2007_58.pdf
3. https://static1.squarespace.com/static/56ff90f3d51cd4118179ec49/t/571da42a45bf21e6a9e47e8b/1461560366359/CT01.pdf
4. http://www.datatronics.com/pdf/ct319.pdf

This footprint will fit all the above 3 parts and it doesn't violate any KLC clauses. Here is an image:
![image](https://cloud.githubusercontent.com/assets/1936989/21078542/a09216c8-bf28-11e6-87ed-f4cd2671ae3a.png)

While I tried to pick a footprint name that is descriptive and can be used to build off of in the future for similar parts, it is pretty ugly. If there is a better footprint name please let me know.

I understand if this isn't the right library, so please tell me if this should go somewhere else. The existing transformer libraries are either for through hole or for specialized transformer type/vendor, so this seemed like the next best module library.

evanshultz committed:
Create Transformer_current_SMD_8.4x7.2mm.kicad_mod

jkriege2 commented:
Hi!

First of all: Seasonal Greetings :-)))))

About the footprint: Nice work so far. Some things are still missing:
1. please move REFDES+NAME text from under the device
2. Please add device outline on F.Fab
3. Name is fine for me (maybe `_current_` --> `_CurrentSense_`), but the lib should be `Choke_SMD` (I think that's the closest fit with the current libs ... and we also have transofrmers in `Choke_Toroid_ThroughHole`)

Best,
JAN

evanshultz committed:
Move from this lib

evanshultz commented:
Happy Holidays to you too!!

I deleted the part, which means this PR can be closed, and re-submitted the patch at https://github.com/KiCad/Choke_SMD.pretty/pull/7.

To address the issues:
1. Done
2. Done. I used 5x6.5mm since the core is 5mm square and the clips are around 1mm but can vary.
3. Renamed to `Choke_SMD_CurrentSense_8.4x7.2mm.kicad_mod`.

jkriege2 commented:
closed as requested
